core/src/main/java/io/questdb/griffin/engine/functions/window/MinDoubleWindowFunctionFactory.java

```diff
@@ -42,9 +42,9 @@
 import io.questdb.std.ObjList;
 
 public class MinDoubleWindowFunctionFactory extends AbstractWindowFunctionFactory {
+    public static final MaxDoubleWindowFunctionFactory.DoubleComparator LESS_THAN = (a, b) -> Double.compare(a, b) < 0;
```

Comment:
Off-topic question: shouldn't we use `Numbers#compare(double, double)` here and in all other double comparator cases? With this comparator window function and "normal" order by will behave differently.

Reply:
good spot, fixed in https://github.com/questdb/questdb/pull/5300/commits/3c891b637c1a21d111fd2cf31361365a90e44a42